photobooth/services/processingservice.py

```diff
@@ -390,9 +390,8 @@ def on_exit_captures_completed(self):
         logger.info("exit job postprocess, adding items to db")
 
         for item in self.model._confirmed_captures_collection:
-            if item.visible:
```

Comment:
Did you double check that? I had to add this because otherwise the items would still be visible in the frontend initially. Only after a forced refresh, the hidden items disappeared from the gallery.

Reply:
it's getting wild... fixed here: 7fa7021